packages/server/src/api/leaderboard/graph.js

```diff
@@ -1,65 +0,0 @@
-import { responses } from '../../responses'
-import * as cache from '../../cache'
-import config from '../../config/server'
-
-export default {
-  method: 'GET',
-  path: '/leaderboard/graph',
-  requireAuth: false,
-  schema: {
-    querystring: {
-      type: 'object',
-      properties: {
-        division: {
-          type: 'string',
-          enum: Object.keys(config.divisions),
-        },
-        limit: {
-          type: 'integer',
-          minimum: 1,
-          maximum: config.leaderboard.graphMaxTeams,
```

Comment:
Currently, runtime schemas are missing important properties derived from config such as this. `makeFastifyRoute` should allow deepmerging on the schema using the third argument to enable this AJV-based parameter validation.
Routes must also be checked manually to make sure none of these types of constraints made it in to the static `@rctf/api-types` sources.

Reply:
I believe I have covered everything...